test/Microsoft.Extensions.ML.Tests/Microsoft.Extensions.ML.Tests.csproj

```diff
@@ -10,10 +10,13 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.Transforms\Microsoft.ML.Transforms.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.StandardTrainers\Microsoft.ML.StandardTrainers.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFrameworkCommon\Microsoft.ML.TestFrameworkCommon.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
```

Comment:
>TestFramework [](start = 47, length = 13)

There was a change recently to move some functionality out of Microsoft.ML.TestFramework and into a separate project Microsoft.ML.TestFrameworkCommon. I don't remember what was the reason for this change (I think it had something to do with the nightly build test project).
Anyway, if all test classes need to inherit from `BaseTestClass`, shouldn't we move that class to Microsoft.ML.TestFrameworkCommon?

Reply:
BaseTestClass has reference to this Contracts which is internal class of Microsoft.ML.Core and referenced by TestFramework by bestfriend. 
I don't know why this Contracts class is in src folder not test folder but that can be a different issue.

---
In reply to: [381876287](https://github.com/dotnet/machinelearning/pull/4858#discussion_r381876287) [](ancestors = 381876287)